src/cryptography/hazmat/primitives/serialization/base.py

```diff
@@ -81,3 +81,26 @@ def __init__(self, password):
 @utils.register_interface(KeySerializationEncryption)
 class NoEncryption(object):
     pass
+
+
+@six.add_metaclass(abc.ABCMeta)
+class CertificateSerializationEncryption(object):
+    pass
+
```

Comment:
We should be able to use the existing ABC and implementations.

Reply:
I'll do that, but I though the work _key_ in _KeySerializationEncryption_ was there for a reason, so I didn't want to specify certificate serialization encryption with it.